src/Core/src/Platform/iOS/MauiWebViewUIDelegate.cs

```diff
@@ -31,14 +57,16 @@ public override void RunJavaScriptConfirmPanel(WKWebView webView, string message
 		}
 
 		public override void RunJavaScriptTextInputPanel(
-			WKWebView webView, string prompt, string defaultText, WKFrameInfo frame, Action<string> completionHandler)
+			WKWebView webView, string prompt, string? defaultText, WKFrameInfo frame, Action<string> completionHandler)
 		{
+			// TODO the okAction and cancelAction were already taking null when I removed `#nullable disable`
+			// and it doesn't seem to have been causing issues so leaving for now
 			PresentAlertController(
 				webView,
 				prompt,
 				defaultText: defaultText,
-				okAction: x => completionHandler(x.TextFields[0].Text),
-				cancelAction: _ => completionHandler(null)
+				okAction: x => completionHandler(x.TextFields[0].Text!),
+				cancelAction: _ => completionHandler(null!)
```

Comment:
???

Maybe the delegate should be a nullable `string?`?

Reply:
AFAIK because it's an override so I can't change the delegate.  The iOS SDK would have to change it. 

If I set it to `Action<string?>` then I get a compile error `Nullability of reference types in type doesn't match overidden member`

Reply:
oh dear :( This sounds like a wrong API. Let me open an issue for that.